torch/csrc/profiler/python/init.cpp

```diff
@@ -122,19 +135,32 @@ void initPythonBindings(PyObject* module) {
         return py::reinterpret_borrow<py::object>(thp_device);
       });
 
-  py::class_<ExtraFields<EventType::TorchOp>>(m, "_ExtraFields_TorchOp")
-      .def_readonly("inputs", &ExtraFields<EventType::TorchOp>::inputs_)
-      .def_readonly(
-          "allow_tf32_cublas",
-          &ExtraFields<EventType::TorchOp>::allow_tf32_cublas_);
+  using torch_op_t = ExtraFields<EventType::TorchOp>;
+  py::class_<torch_op_t>(m, "_ExtraFields_TorchOp")
+      .def_readonly("inputs", &torch_op_t::inputs_)
+      .def_readonly("scope", &torch_op_t::scope_)
+      .def_readonly("sequence_number", &torch_op_t::sequence_number_)
```

Comment:
When looking in the codebase, I didn't find a docstring documenting what this `sequence_number` is supposed to hold. Could you add a docstring to where the C++ field is declared to explain the fields and their properties in a separate diff?